### ISSUE TITLE
Doesn't Support Running on arm64 Self-Hosted Runners

### ISSUE THREAD

**Issue by timnolte** [WillAbides/setup-go-faster#29]
So it appears that the way this is running/installing Go that it isn't compatible with arm64. I simply swapped out this runner for the one that says it's better than and the Go installation completely fails.

```
./src/semver-select: line 33: /tmp/runner/_actions/WillAbides/setup-go-faster/v1.12.0/bin/semver-select: cannot execute binary file: Exec format error
./src/semver-select: line 33: /tmp/runner/_actions/WillAbides/setup-go-faster/v1.12.0/bin/semver-select: Success
/tmp/runner/wp-dev-container/setup-go-faster/tmp
src/install-go: line 43: /opt/hostedtoolcache/go/1.19.13/x64/bin/go: cannot execute binary file: Exec format error
```

**Comment by WillAbides** [comment]
You're right. It assumes amd64 architecture because that's what all of the hosted runners used when I created it and I didn't foresee demand for other architectures.

I'll look at adding it support for arm64.

**Comment by WillAbides** [comment]
Support for arm64 is added in [v1.13.0](https://github.com/WillAbides/setup-go-faster/releases/tag/v1.13.0).

I don't have access to an arm64 machine, so I haven't tested it. If you have a chance to try it, I would appreciate if you comment here letting me know how it goes.

**Comment by timnolte** [comment]
@WillAbides thanks I'll give this a test later this week.
